--- src/player/Car.ts
+++ src/player/Car.ts
@@ -84,7 +84,7 @@
     // Move with collisions
     const moveVector = this.velocity.scale(dt);
     // Add fake gravity
-    moveVector.y = -0.5;
+    moveVector.y = 0;
     
     this.mesh.moveWithCollisions(moveVector);
 
--- src/ai/PoliceCar.ts
+++ src/ai/PoliceCar.ts
@@ -93,7 +93,7 @@
     
     // Move
     const moveVector = this.velocity.scale(dt);
-    moveVector.y = -0.5;
+    moveVector.y = 0;
     
     this.mesh.moveWithCollisions(moveVector);
     this.mesh.position.y = 0.4;
